waflyctl.go

```diff
@@ -50,29 +50,30 @@ var (
 
 // TOMLConfig is the applications config file
 type TOMLConfig struct {
-	Logpath       string
-	APIEndpoint   string
-	Tags          []string
-	Publisher     []string
-	Action        string
-	Rules         []int64
-	DisabledRules []int64
-	Owasp         owaspSettings
-	Weblog        WeblogSettings
-	Waflog        WaflogSettings
-	Vclsnippet    VCLSnippetSettings
-	Response      ResponseSettings
-	Prefetch      PrefetchSettings
+	Logpath            string
+	APIEndpoint        string
+	Tags               []string
+	Publisher          []string
+	Action             string
+	Rules              []int64
+	DisabledRules      []int64
+	Owasp              owaspSettings
+	Weblog             WeblogSettings
+	Waflog             WaflogSettings
+	Vclsnippet         VCLSnippetSettings
+	AdditionalSnippets map[string]VCLSnippetSettings
+	Response           ResponseSettings
+	Prefetch           PrefetchSettings
 }
 
 // Backup is a backup of the rule status for a WAF
 type Backup struct {
 	ServiceID string
 	ID        string
 	Updated   time.Time
-	Disabled  []string
-	Block     []string
-	Log       []string
+	Disabled  []int64
```

Comment:
Additionally, maybe it'd be a good idea to sync format produced by `--backup` with `waflyctl.toml` one (`disabled` vs `disabledrules`)?

Reply:
The waflyctl config toml uses ints for rule IDs, thus the switch for back-up functionality. Without this, you end up having the convert the back-up output for use in a configuration file

Reply:
I totally get it - we encountered exactly that problem during WAF training call :)  

Just wondering if ints do make sense here. On one hand there's no math involved, so I'd opt for strings, but on the other hand ints would be much easier to manage in TOML file (essentially you can forget about enclosing `"` around every rule ID).

Reply:
Hey Jakub - that's _exactly_ it. Parity between the configuration file format and the back-up file format = path of least (or lesser) resistance when you've got that I-need-to-restore feeling :-) 

The waflyctl toml file uses ints (I think your first question was asking 'why' we use ints for rule ids? I can't answer that, but it may be the case that the underlying schema uses ints for rule IDs, so perhaps there's an element of type safety)